libnavigator/build.gradle

```diff
@@ -31,6 +31,7 @@ android {
 
 dependencies {
     implementation(project(':libnavigation-base'))
+    implementation dependenciesList.mapboxCommon
```

Comment:
Why this dependency was added 🤔? Do you remember @Lebedinsky? I've removed it and didn't run into any issues cc @LukasPaczos

Reply:
Refs. https://github.com/mapbox/mapbox-navigation-android/pull/2774#discussion_r408739288

Reply:
We can remove it
I added it before Sensor classes was refactored by Kyle, and it was needed…. but after Kyle’s refactoring it is useless anymore. Great catch, Pablo 👍